rando_modules/logic.py

```diff
@@ -821,7 +822,6 @@ def _algo_assumed_fill(
     print("Placing progression items...")
     #Place progression items, both key and replenishable
     pool_combined_progression_items = pool_progression_items + pool_misc_progression_items
-    random.shuffle(pool_combined_progression_items)
```

Comment:
This turns of shuffling the item pool, was this intended?

Reply:
Nope, big blunder, unnoticed local edit that got commited by mistake. Reverted.